app/controllers/api/v1/media_controller.rb

```diff
@@ -25,6 +25,15 @@ def update
     render json: @media_attachment, serializer: REST::MediaAttachmentSerializer, status: status_code_for_media_attachment
   end
 
+  def destroy
+    @media_attachment = current_account.media_attachments.find(params[:id])
```

Comment:
I opted for doing this rather than modifying `set_media_attachment` because I didn't want to modify the behavior of `PATCH/PUT /api/v1/media/:id`, which currently would return a 404 if the media attachment is currently in use.

Arguably that API should be changed to return an "in usage" error, but I wanted to avoid a breaking API change.